my_static_website_infra/bucket_configuration.sh

```
@@ -1,5 +1,23 @@
 BUCKET_NAME="my-static-website-04"
-ENDPOINT=
+ENDPOINT=https://localhost.localstack.cloud:4566
+
+echo "Starting S3 Bucket Configuration for Static Website Hosting"
+
+# Set AWS CLI to use LocalStack
+export AWS_PROFILE=localstack
+
+# Enable static website
+aws --endpoint-url=$ENDPOINT s3 website s3://my-website-04/
+    --index-document index.html
```

Shellcheck (0.10.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)